tests/test_cmd_signature.py

```diff
@@ -882,6 +882,59 @@ def test_sig_describe_1_hp(c):
         assert cleaned_line in expected_output
 
 
+def test_describe_bug_905():
+    # Explicitly test https://github.com/dib-lab/sourmash/issues/905
+    with utils.TempDirectory() as location:
+        testdata1 = utils.get_test_data('protein_905.sig')
+
+        status, out, err = utils.runscript('sourmash',
+                                           ['signature', 'describe',
+                                            testdata1],
+                                           in_directory=location)
+        expected_output == """== This is sourmash version 3.2.2. ==
```

Comment:
```suggestion
        expected_output = """
```
there is an extra `=` here, and since versions change all the time we don't want that in the test